app/models/unit.rb

```diff
@@ -23,6 +23,7 @@ class Unit < ApplicationRecord
   has_many :users, through: :employees
   has_many :positions, through: :duties
   has_many :administrative_functions, through: :duties
+  has_many :agendas, dependent: :nullify
```

Comment:
`dependent:nullify` kullanıyorsan bu senaryoyu kodun diğer yerlerinde de işletmen lazım. Örneğin aşağıda view'larda `agenda.unit.name` satırı var. Eğer bir unit silinirse, agenda'nın unit'i bulunamadığı için bu sayfa erişilemez olur, hataya düşer.